common/shell-local/config.go

```diff
@@ -19,14 +20,17 @@ import (
 type Config struct {
 	common.PackerConfig `mapstructure:",squash"`
 
+	shell.Provisioner `mapstructure:",squash"`
```

Comment:
I believe this also embeds `common.PackerConfig` so we might be able to get rid of line 21

Reply:
Ah, I'll check.

Reply:
yep, you're right.